src/modules/oauth/containers/AuthContainer.js

```diff
@@ -0,0 +1,130 @@
+import React, { Component } from 'react'
+import { Button, ButtonGroup, ButtonToolbar,Card, CardBody, CardHeader, CardFooter, CardGroup, Col, Container, Row } from 'reactstrap';
+import { Link } from 'react-router-dom';
+import OauthPopup from 'react-oauth-popup';
+import queryString from 'query-string'
+
+
+class AuthContainer extends React.Component {
+
+    constructor(props) {
+      super(props);
+      this.AuthService =  this.props.app.locateService("AuthService");
+
+
+      this.state = {
+        buttons:[]
+      };
+
+      const TLdomain = "https://via.teskalabs.com/seacat-auth";
```

Comment:
This is too specific to be part of the  generic technology.

Reply:
This will be factored out, url will be in specific authMethods.